src/templates/_index.twig

```diff
@@ -107,6 +106,21 @@
                     </div>
                 </div>
             </div>
+            {% if updates %}
+                <div class="front" style="position: relative;">
+                    <div class="pane notice-pane">
+                        <a class="right icon delete close-notice" title="Close" style="margin-top: -1%; padding-left: 1%; font-size: 1.2rem"></a>
+                        <div class="body" style="width:100%; text-align: center; display: flex; align-items: center; flex-direction: column;">
+                            <td style="text-align:center;">
+                                <span>There is an update available for Translations.
+                                    <a class="go nowrap" href="utilities/updates"> View details
+                                    </a>
+                                </span>
+                            </td>
+                        </div>
+                    </div>
+                </div>
+            {% endif %}
```

Comment:
@shnsumit we need to separate these notifications.

<img width="1709" alt="image" src="https://user-images.githubusercontent.com/11489608/121458819-4e746f00-c967-11eb-8a65-50fe9abec73f.png">

Removing one removes both and sets the display cookie.
<img width="1786" alt="image" src="https://user-images.githubusercontent.com/11489608/121458851-60eea880-c967-11eb-995a-c836852c4b98.png">

No need to set an "x" action or cookie for the update notice.